ruma-events-macros/src/event_enum.rs

```diff
@@ -202,6 +206,134 @@ fn marker_traits(ident: &Ident) -> TokenStream {
     }
 }
 
+fn accessor_methods(ident: &Ident, variants: &[Ident]) -> syn::Result<TokenStream> {
+    let any_content = ident.to_string().replace("Stub", "").replace("Stripped", "");
+
+    let content_enum = Ident::new(&format!("{}Content", any_content), ident.span());
+
+    let origin_server_ts = if ident == "AnyBasicEvent"
```

Comment:
If we're not going to have one big macro invocation, you should refactor these ident comparisons, maybe move them into helper functions as a first step.

Reply:
How do you feel about also using `const`s for the comparisons
```rust
pub const ANY_STATE_EVENT: &str = "AnyStateEvent";
pub const ANY_SYNC_STATE_EVENT: &str = "AnyStateEventStub";
```
that way name changes are less work?

For the helpers something like
```rust
fn has_room_id(ident: &Ident) -> bool {
    ident == event_names::ANY_MESSAGE_EVENT
        || ident == event_names::ANY_STATE_EVENT
        || ident == event_names::ANY_EPHEMERAL_EVENT
}
```

Reply:
Seems fine. For me the much bigger reason to do this would be preventing typos, rather than making name changes easier. Either way, I like it.